flightplan/flightplan.d.ts

```diff
@@ -0,0 +1,201 @@
+// Type definitions for flightplan v0.6.9
+// Project: https://github.com/pstadler/flightplan
+// Definitions by: Borislav Zhivkov <https://github.com/borislavjivkov>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare var flightplan: FlightPlanInterfaces.IFlightplan;
+
+declare module FlightPlanInterfaces {
+  interface ICommandOptions {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.